content/blog/ggplot-3-5-0-guides/index.Rmd

```diff
@@ -0,0 +1,895 @@
+---
+output: hugodown::hugo_document
+
+slug: ggplot-3-5-0-guides
+title: A guide to guides
+date: 2023-12-20
+author: Teun van den Brand
+description: >
+    The new 3.5.0 release of ggplot2 implements a new guide system. Read on
+    to find out what is new.
+
+photo:
+  url: https://unsplash.com/photos/n6vS3xlnsCc
+  author: Kelley Bozarth
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [ggplot2]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're tickled pink to announce the release of [ggplot2](https://ggplot2.tidyverse.org) 3.5.0. 
+ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics. 
+You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("ggplot2")
+```
```

Comment:
I don't think we should repeat this for the second blog post. Maybe something along the line of

> We're super happy to release ggplot2 3.5.0. This is the second blog post Jada Jada Jada, link to the other blog post etc